shared-libs/settings/test/index.js

```diff
@@ -1,91 +0,0 @@
-const chai = require('chai');
```

Comment:
This is because I changed the name from `index.js` to `index.spec.js`, I thought git would show it as `rename` 🤔

Reply:
I've also refactored the unit tests